src/Components/PresentationalComponents/TableColumns/RemediationColumn.js

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment } from 'react';
+import PropType from 'prop-types';
+import { AnsibeTowerIcon } from '@patternfly/react-icons';import { Tooltip } from '@patternfly/react-core';
+import { FormattedMessage } from 'react-intl';
+import messages from '../../../Messages';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+
+const RemediationColumn = ({ fixable }) => {
+    return (
+        <Fragment>
+            {fixable === true && <Fragment><AnsibeTowerIcon/> <FormattedMessage {...messages.playbook} /></Fragment>}
+            {fixable === false && <FormattedMessage {...messages.manual} />}
```

Comment:
Ahhh I find it dangerous to have `false` and `null` having different semantics while we don't have Typescript, but let's keep it like this and then we can refactor it later
> Make it work, make it right, make it fast

Reply:
for that reason, I explicitly checked the type  `fixable === false` and not just `fixable && <FormattedMessage....`.  Also, I don't know how Typescript will help us here other than stating the type. There are two different states

1. false -  it means that the variable has been declared AND has been defined as `false`
2. null - it means that the variable has been declared BUT has not been defined yet - aka empty 

and by not stating `isRequired` as prop-type I allow both